### PR TITLE
removed 'scary' words from wordlist

### LINKED ISSUE #1541: "Treason" as a passphrase word option for Sources? #fail
https://docs.securedrop.org/en/latest/journalist.html

Super random: but, the third-down screenshot I came upon earlier today, and it shows a created passphrase with the word "treason" in it. Imagining what a n00b Government Accountability type leaker might think upon seeing that—and all I can think, is task abandonment, buh-bye.

Words such as "treason" and "traitor" or "prison" or "murder" or "stalking" or "jail" or "guilty" or "betrayal" or "deported" or "exile" etc. Would it be possible to assemble a list of likely trigger words to be blocked from inclusion in passphrases presented to Sources?

### PR DESCRIPTION
**Pull request by heartsucker**
Fixes #1541

### CODE REVIEW

File: securedrop/wordlist [hunk 1]
```diff
@@ -5079,7 +4960,6 @@ ranger
 ranging
 ranked
 ranking
-ransack
```
Comment (redshiftzero) [inline securedrop/wordlist]: Think this can go back in

**Comment by redshiftzero** [comment]
(Note: removing 178 words from the list reduces entropy from 12.89 to 12.85 bits per word - a negligible change - so security-wise I think this is good to merge)

File: securedrop/wordlist [hunk 2]
```diff
@@ -1012,7 +989,6 @@ chihuahua
 childcare
 childhood
 childish
-childless
```
Comment (heartsucker) [inline securedrop/wordlist]: My reasoning on this one would a that a whistleblower could lose custody of a child as a result of their actions, and this word might be "triggering" in that regard.

Comment (redshiftzero) [inline securedrop/wordlist]: Hmm fair point, feel free to leave in

File: securedrop/wordlist (hunk 1 above)
Comment (heartsucker) [inline securedrop/wordlist]: I pulled this one because leak -> police raid -> ransack seemed like too easy a jump. Could put it back it still.

**Comment by redshiftzero** [comment]
Looks good, merging!

**Pull request merged**